datasets/InputData_Mixed.csv

```diff
@@ -0,0 +1,7 @@
+timestamp,Outside Air Temperature (C),Outside Air Relative Humidity (1-100%),Outside Air Humidity Ratio (-),Zone Thermostat Requested Sensible Heat Transfer Rate (kW),Supply Air Temperature (C),Supply Air Relative Humidity (0-100%),Supply Air Humidity Ratio,Water Use (Liters/min),System Electric Power (kW),Operating Mode,Supply Air Mass Flow Rate (kg/s),Supply Air Volume Flow Rate (m3/s),Sensible System Cooling Capacity (kW),Latent System Cooling Capacity (kW),Total System Cooling Capacity (kW)
```

Comment:
This is input data for the unit test, we provide the inputs the main calculation uses, and the expected outputs. The data relates to the system that is being tested in the unit test.  The reason its a file is that we use this unit test for our validation exercise, that uses 40k lines of real data. The InputData_Mixed.csv file is a small example of that data.  So, Im not sure of where else this should be.